crates/fluvio/Cargo.toml

```diff
@@ -50,6 +51,8 @@ quantities = { version = "0.10.0", features = [
 hdrhistogram = { version = "7.5.0", optional = true }
 strum = { version = "0.24", features = ["derive"], optional = true }
 num-traits = { version = "0.2", optional = true }
+opentelemetry = { version = "0.18.0", optional = true, features = ["metrics", "rt-async-std"] }
```

Comment:
Maybe we should introduce a separate crate (e.g. `fluvio-opentelemetry`) where we can hide these 3rd party deps and also some common methods. We will have many crates having such dependencies, with one crate it would be easier to control all deps and features.

Reply:
Agreed to look into this improvement in the future